packages/create-sitecore-jss/src/templates/angular/scripts/generate-config.ts

```diff
@@ -50,7 +54,14 @@ const config = {};\n`;
 
   // Set base configuration values, allowing override with environment variables
   Object.keys(config).forEach((prop) => {
-    configText += `config.${prop} = process.env.${constantCase(prop)} || "${config[prop]}",\n`;
+    if (prop === 'siteName') {
```

Comment:
Similar to vue comment - do you think it can be better if we just use process.env values as default ones for siteName on line 19? This would simplify the code since the extra clause won't be needed